### PR TITLE
✨ [Feature #504]: Add umami in the tech stack page

### LINKED ISSUE #504: ✨ [Feature request]: Add umami in the tech stack page
As part of this PR, we have added self-hosted analytics for our platforms, and we should add the umami to the tech stack page.

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

I worked on adding umami in the tech stack page

Fixes #504 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@atapas @koustov , PR is ready for review.

File: src/common/techstack/TechStackInfo.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
   SiNodedotjs,
 } from "react-icons/si";
 import NHost from "images/nhost.svg";
+import Umami from "images/umami.svg";
```
Comment (koustov) [inline src/common/techstack/TechStackInfo.js:18]: The image needs to be little bigger in order to be in sync with others.  Refer https://analytics.rip/images/tools/umami.svg

Also can we give it a grayish look? On hover show the actual color

**Review by koustov — CHANGES_REQUESTED** [review]
a tiny observation

File: src/common/techstack/TechStackInfo.js (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/common/techstack/TechStackInfo.js:18]: Done, I have Changed the Image.

Comment (koustov) [reply src/common/techstack/TechStackInfo.js:18]: yeah that'sd correct however my point was to make initial one with less opacity and hover will make the `opacity: 1`

**Review by koustov — APPROVED** [review]
lgtm

**Comment by Sachin-chaurasiya** [comment]
@atapas , Awaiting for your approval.

**Review by atapas — APPROVED** [review]

**Pull request merged**